Backend/UnitTesting/UnitTest1.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;

```

Comment:
This should be more meaningful. Make this test one of your services or such, even if it's a sanity check (true == true)